include/net/nrf_cloud_logs.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef NRF_CLOUD_LOGS_H_
+#define NRF_CLOUD_LOGS_H_
+
+/** @file nrf_cloud_logs.h
```

Comment:
Yeah, I understand your point.  I was following the same style as I used for the alerts feature, where the public include is `nrf_cloud_alerts.h` but the function prefix is `nrf_cloud_alert_`.

Reply:
Internally we've been calling these the "alerts and logs services", so using `nrf_cloud_alerts.h` and `nrf_cloud_logs.h` makes sense to me.  But as a prefix, the plural form `nrf_cloud_logs_send()` seems strange compared to `nrf_cloud_log_send()`, since this sends just one log line.

Reply:
I agree that `nrf_cloud_logs_send()` is not good.
I'd still name the file nrf_cloud_log.h because the filename and function prefix is more important for me than aligning with the written name, in case you want to call it "logs services" instead of "log services".
Currently, it's not nice and consistent but it's your call.

I understand "log services" may not be nice because you have "alerts services". However, I'd probably change alert API to "alert services". Not a big change in the documentation if any. Not perfect either as the header file would be `nrf_cloud_alerts.h` but renaming that because of this would be too much.
BTW, the only time service is mentioned in [Alerts document](https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-10266/nrf/libraries/networking/nrf_cloud_alerts.html) is: `The nRF Cloud Alert service`.
I can see "Alerts feature", etc. are used a bit in the documentation.